krux/cli.py

```diff
@@ -108,7 +110,7 @@ def __init__(self, name, parser=None, logger=None, lockfile=False):
         self.add_cli_arguments(self.parser)
 
         ### and parse them
-        self.args = self.parser.parse_args()
+        self.args = self.parser.parse_args(args=cli_args)
```

Comment:
`self.add_cli_arguments(self.parser)` actually adds the available options, rather than adding strings passed in from the command line.
This line takes custom arguments in place of `system.argv` for the parser.